docs/_docs/live-update/running-offline.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Running Offline'
+layout: page
+parent: Live Update
+nav_order: 2
+---
+
+# Saving cached Figma files for offline use
+
+This method will allow you to add a pre-fetched Figma file to your DesignCompose app's assets. This allows the app to run and display your design without a network connection and without a Figma authentication token set.
+
+1. Launch the app as normal in an emulator, set the Figma Token i needed, and allow the Figma file to be fetched and displayed
+
+1. Create a directory in your app's `src/main` directory named `assets/figma`. (See `reference-apps/helloworld/src/main/assets/figma` for an example)
+
+1. Open a terminal and navigate to that directory. On way to do this is to right-click the directory in Android Studio and select **Open In -> Terminal**
```

Comment:
'On' -> 'One'

Reply:
I'm really starting to wonder if it's time for a new keyboard...